src/build/build.ts

```diff
@@ -160,6 +162,10 @@ export function build(options?: BuildOptions, config?: ProjectConfig): Promise<a
       .then((depsIndex) => {
         let unbundledDeps = analyzer.allFragments
             .concat(Array.from(depsIndex.depsToFragments.keys()));
+            
+        let fullDeps = Array.from(depsIndex.fragmentToFullDeps.values());
+        let scriptDeps = new Set<string>();
```

Comment:
Instead of `scriptDeps`, this should be `scriptAndStyleDeps` and include any external styles.